README.md

```diff
@@ -99,20 +99,79 @@ okta_client = OktaClient()
 
 Okta allows you to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
 
-This SDK supports this feature (OAuth 2.0) only for service-to-service applications. Check out [our guides][oauth-guides] to learn more about how to register a new service application using a private and public key pair.
+This SDK supports this feature (OAuth 2.0) only for service-to-service applications. Check out [our guides](https://developer.okta.com/docs/guides/implement-oauth-for-okta-serviceapp/overview/) to learn more about how to register a new service application using a private and public key pair.
 
 When using this approach you won't need an API Token because the SDK will request an access token for you. In order to use OAuth 2.0, construct a client instance by passing the following parameters:
 
 ```py
 from okta.client import Client as OktaClient
 config = {
-    'orgUrl': 'https://test.okta.com',
+    'orgUrl': 'https://YOUR_DOMAIN',
```

Comment:
change to `https://{yourOktaDomain}` to match the standard

Reply:
done